app/frontend/src/components/form/form.test.js

```diff
@@ -0,0 +1,37 @@
+import { disableInputs, enableInputs } from './form';
+
+describe('form', () => {
+  document.body.innerHTML = '<input id="test-text-input" type="text"/>';
+  const textField = document.querySelector('#test-text-input');
```

Comment:
getElementById is always preference to select DOM elements - if its possible to use it in a sensible way

Reply:
`document. getElementById('test-text-input');`